### PR TITLE
Docstring cleanups.

### PR DESCRIPTION
**Pull request by anntzer**
Mostly D401 (imperative mood) fixes, and a few other things.

## PR Checklist

- [ ] Has pytest style unit tests (and `pytest` passes).
- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check).
- [ ] New features are documented, with examples if plot related.
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] Conforms to Matplotlib style conventions (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).
- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [ ] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).

### CODE REVIEW

File: lib/mpl_toolkits/axisartist/axisline_style.py [hunk 1]
```diff
@@ -9,9 +9,7 @@
 
 class _FancyAxislineStyle:
     class SimpleArrow(FancyArrowPatch):
-        """
-        The artist class that will be returned for SimpleArrow style.
-        """
+        """The artist class that will be returned for SimpleArrow style."""
```
Comment (story645) [inline lib/mpl_toolkits/axisartist/axisline_style.py:12]:
```suggestion
        """The artist returned when arrow style is SimpleArrow"""
```

File: lib/mpl_toolkits/axisartist/axisline_style.py [hunk 2]
```diff
@@ -69,9 +67,7 @@ def draw(self, renderer):
             FancyArrowPatch.draw(self, renderer)
 
     class FilledArrow(SimpleArrow):
-        """
-        The artist class that will be returned for SimpleArrow style.
-        """
+        """The artist class that will be returned for SimpleArrow style."""
```
Comment (story645) [inline lib/mpl_toolkits/axisartist/axisline_style.py:70]: What triggers this versus returning the SimpleArrow above?

**Review by story645 — APPROVED** [review]
it's better even if my comments are ignored, I'm just confused

**Comment by anntzer** [comment]
Probably the whole axisline_style needs rewording, but let's not do that here.

**Comment by QuLogic** [comment]
@meeseeksdev backport to v3.5.x

**Pull request merged**